packages/app/src/app/store/modules/server/actions.js

```diff
@@ -0,0 +1,20 @@
+import { dispatch } from 'codesandbox-api';
+
+export function restartSandbox({ executor }) {
+  executor.emit('sandbox:restart');
+}
+
+export function restartContainer({ executor }) {
+  executor.emit('sandbox:restart-container');
+}
+
+/**
+ * We need to remove this logic later! We need to move it to the terminal component
```

Comment:
@CompuIves Add `TODO:` in front of this comment so IDE's can track this

```suggestion
 * TODO: We need to remove this logic later! We need to move it to the terminal component
```